### ISSUE TITLE
MueLu: configure error on geminga nightly build (Tpetra not enabled)

### ISSUE THREAD

**Issue by jhux2** [trilinos/Trilinos#10940]
## Bug Report
@trilinos/muelu 

### Description

Build with Tpetra explicitly enabled, https://testing.sandia.gov/cdash/build/11222446

```
CMake Error at packages/muelu/CMakeLists.txt:21 (MESSAGE):
  MueLu requires that if Kokkos is enabled, Tpetra must also be enabled.
```

**Comment by GrahamBenHarper** [comment]
There's a `Trilinos_ENABLE_Tpetra=ON` followed later by a `Trilinos_ENABLE_Tpetra=OFF` flag in the configure flags

**Comment by cgcgcg** [comment]
Closed by #11025
